frontend/src/Pages/Profile.js

```diff
@@ -0,0 +1,95 @@
+import React, { useState, useEffect } from 'react'
+import { useHistory } from 'react-router-dom'
+import { useSelector, useDispatch } from 'react-redux'
+
+import { FEELING_URL } from '../reusable/urls'
+import user from '../reducers/user'
+import { feelingsArray } from '../reusable/constants'
+
+const Page = () => {
+  const [feelings, setFeelings] = useState('')
+  const name = useSelector(state => state.user.username)
+  const accessToken = useSelector(store => store.user.accessToken)
+  const userId = useSelector(store => store.user.userId)
+  const feelingsHistory = useSelector(store => store.user.feelings)
+
+  const dispatch = useDispatch()
+  const history = useHistory()
+
+  useEffect(() => {
+    if (!accessToken) {
+      history.push('/');
+    }
+  }, [accessToken, history]);
+
+  const onButtonClick = (e) => {
+    const options = {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+        'Authorization': accessToken
+      },
+      body: JSON.stringify({ feelings: e.target.value })
+    }
+    fetch(FEELING_URL(userId), options)
+      .then(res => res.json())
+      .then(data => dispatch(user.actions.setFeelings(data.feelings)))
+    setFeelings(e.target.value)
+  }
+
+  const onFeelingsRegistred = (feelings) => {
```

Comment:
I think it would be nice to have a list on your database. it would be easier to manage your app in the future.